foss4gna_24/big-EO-viz.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>the link is messed up here <a href="https://notebooksharing.space" target="_blank">https://notebooksharing.space</a></p><p> </p><p>I don't know if "frozen" is the right way to describe, I would say so the maps and data are embedded in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
Updated as follows - 


### Upload the Notebook
After executing the above cell, you can find the executed notebook in the same directory with the name big-EO-viz-executed.ipynb. You can upload this notebook to the [Notebook Sharing](notebooksharing.space) and get an unique URL for sharing.

Reply:
<p>Frozen should not be in the text.</p><p>"generate ... noteboook ... so that all interactive visualizations are included in a non-python rendering ..."</p>